moovingon_ai/README.md

```diff
@@ -0,0 +1,47 @@
+# moovingon.ai
+
+## Overview
+[moovingon.ai][1] is a platform for cloud operations and NOC management. It consolidates alerts across your observability suite and associates them with automated runbooks for alert and incident remediation. By integrating this, you can utilize the power of Datadog and moovingon.ai for efficient, automated incident management.
+
+moovingon.ai uses Datadog's monitors, logs, and event data for alert correlation and aggregation.
+Key features of this integration include:
+
+1. **Focused Alert Management**: Bring all your Datadog alerts together in moovinon.ai easy-to-view dashboard, giving you simple control and clear visibility.
+2. **Comprehensive incident management**: Log all remediation actions performed in moovingon.ai in Datadog events for compliance and remediation clarity.
```

Comment:
Is this automatically logged? If so, we should be clear: 

"all remediation actions performed in moovingon.ai are sent to Datadog as events for compliance and remediation clarity "